File: cses/introductoryProblems/repetitions.cpp
#include <iostream>
using namespace std;

int main(void)
{
	string s;
	cin >> s;

	int count = 1;
	int length = s.size();

	for (int i = 0; i < length; i++)
	{
		int j = i + 1, tempCount = 1;
		for ( ; s[i] == s[j]; j++)
		{
			tempCount++;
		}
		count = max(count, tempCount);
		i = j - 1;
	}
	cout << count << endl;
}

File: cses/introductoryProblems/repetitions.java
import java.util.Scanner;

public class repetitions
{
	public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);
		String s = sc.nextLine();

		int count = 1;

		for (int i = 0; i < s.length(); i++)
		{
			int j = i + 1, tempCount = 1;
			for ( ; j != s.length() && s.charAt(i) == s.charAt(j); j++)
			{
				tempCount++;
			}
			count = Math.max(count, tempCount);
			i = j - 1;
		}

		System.out.println(count);
	}
}